src/mocks/encryptor.mock.ts

```diff
@@ -16,17 +17,18 @@ const INVALID_PASSWORD_ERROR = 'Incorrect password.';
 const MOCK_HARDCODED_KEY = 'key';
 const MOCK_HEX = '0xabcdef0123456789';
 const MOCK_SALT = 'SALT';
+// eslint-disable-next-line no-restricted-globals
```

Comment:
It feels like we ought to be able to access Node globals in test-related files. Wondering if there is a change to the ESLint config we can make to allow this. I'll do some research.